posts/2021/02/accessibility-whos-responsible.md

```diff
@@ -0,0 +1,130 @@
+and it won’t be in the near future. Any progress is better than no progress, and several people doing even 
+a little work can add up faster than you might think. Besides, there’s a good chance you won’t even have 
+to go out of your way to start improving accessibility. Start by asking questions about a project you are 
+already working on. Is there a recommended way to design and build a component? Is information represented 
+in more than one way? Is everything labeled?  It’s good practice and more sustainable to consider 
+accessibility as a regular part of your process instead of a special side project.
+
+**It’s not a good use of my energy to work on something that only affects a few people!**
+It’s not just a few people. Read what [WHO](https://www.who.int/en/news-room/fact-sheets/detail/disability-and-health) 
+and the [CDC](https://www.cdc.gov/ncbddd/disabilityandhealth/infographic-disability-impacts-all.html) have 
+to say about the number of people with disabilities.
+
+**I don’t want to make abled people’s experience different than it already is!**
+Depending on what you are doing, the changes might not be active or noticeable unless assistive technologies 
+or accessibility features are being actively used. And in many cases, accessibility features improve the 
+experience for all users and not just those they were designed for. Even if you aren’t convinced, I’d encourage 
+you to ask yourself why creating the user experience you want and making that experience accessible are mutually 
+exclusive. What are people missing out on if they can’t use your product? What are you missing out on if they 
+can’t use your product?
+
+### What could responsibility be like?
+With JupyterLab, it was just a matter of a few people who were willing to say they were tired of waiting and able 
+to spend time both learning what needed to be done as well as doing it. Speaking for myself, I did not come in as 
+an expert or with undivided obligations or even someone with all the skills to make changes that are needed. I 
+think this is important to note because it seems to me that it could have just as easily been other members of 
+the community in my position given similar circumstances. 
+
+Our first step in taking responsibility was setting up regular time to meet so we can check in and help 
+one another. Then we set reasonable goals and scoped the work: we decided to focus on JupyterLab rather 
+than multiple projects at once, address [WCAG 2.1 standards](https://www.w3.org/TR/WCAG21/) in parts of JupyterLab we were already 
+working on, and follow up on past work that other community members began. This is just the beginning, 
+but I hope it was a helpful peek into the process we are trying out.
+___
+
+*This is part of a series of blogs around making JupyterLab more accessible. You can read the 
+[whole series here](https://labs.quansight.org/categories/JLabA11y).*
```

Comment:
This can be ended with a bit more punch - you are already linking to the series. So maybe add a cliffhanger like:

Join me for the next blogpost in this series where I'll be taking about.... or something down these lines